### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by Varun-28**
Done with docs page which is my *1st page after* the home page (landing page). 
Review it !
docs

### CODE REVIEW

File: css/component_styles/getStarted.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    width: 80%;
+    height: auto;
+    margin: 2rem auto;
+}
+
+.horizontal-line2{
+    display: block;
+    border-bottom: solid 0.1rem var(--secondary-color);
+    width: 100%;
+    padding: 0 !important;
```
Comment (bhanuprakashj) [inline css/component_styles/getStarted.css]: good to not have !important unless it's the last resort

File: css/component_styles/getStarted.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    column-gap: 1rem;
+    overflow: auto;
+}
+
+::-webkit-scrollbar {
+    display: none;
+}
+
+.drawer ul li {
+    padding: 0.5rem;
```
Comment (Hamdrive) [inline css/component_styles/getStarted.css]: Keep small units in `px` rather than `rem`

**Review by Hamdrive — COMMENTED** [review]
changes recommended, overall looks good

File: css/component_styles/getStarted.css (hunk 1 above)
Comment (Varun-28) [reply css/component_styles/getStarted.css]: thanks for the correction, i did removed !important and fixed the issue

File: css/component_styles/getStarted.css (hunk 2 above)
Comment (Varun-28) [reply css/component_styles/getStarted.css]: yeah ! thanks for the suggestion, i did changed that

**Comment by Varun-28** [comment]
✔️Done with the get started component (1st page after my landing page)
✔️Made changes as per PR reviews

**Comment by Varun-28** [comment]
Closed by mistake, now re-opened again to merge it with the dev branch

**Pull request merged**